__tests__/app.test.js

```diff
@@ -115,6 +115,37 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("status 200: responds with a comments array for the given article_id in which the comment objects have the following properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(8);
+        comments.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test("status 404: responds for invalid paths in comments", () => {
+    return request(app)
+      .get("/api/articles/2/coments")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Invalid path");
+      });
+  });
+});
```

Comment:
Also need tests for:
- invalid article ID (/api/articles/banana/comments)
- article not found (/api/articles/9999/comments)
- valid article that doesn't have any comments associated with it (should be a OK with an empty arr)